jupyter-archimedes.opam

```diff
@@ -18,6 +18,6 @@ depends: [
   "ocaml" {>= "4.04.0"}
   "dune" {build & >= "1.0.0"}
   "jupyter"
-  "cairo2" {< "0.6"}
+  "cairo2"
```

Comment:
Add also a lower bound on this (it would likely fail due to missing label warning if you use it with old cairo)